### PR TITLE
Third milestone: Created contact form

### PR DESCRIPTION
**Pull request by Khawanin**
### Portfolio:
In this milestone we:

- created contact form
- used flexbox to align the items
- used form validations
- used linters to beautify and validate the code syntax

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,195 +1,238 @@
...
+          <input class="email" type="email" name="email" placeholder="Please enter your email" required>
+        </li>
+        <li>
+          <textarea class="message" name="message" maxlength="500" placeholder="Write your message here"
+            required></textarea>
+        </li>
+        <li>
+          <button class="submit" type="submit">Get it touch</button>
+        </li>
+      </ul>
```
Comment (prolajumokeoni) [inline index.html:233]: - [ ] Kindly delete the `ul` and `li` tag wrapping the elements in the form 👍🏿

File: index.html [hunk 2]
```diff
@@ -1,195 +1,238 @@
...
+        </li>
+        <li>
+          <input class="email" type="email" name="email" placeholder="Please enter your email" required>
+        </li>
+        <li>
+          <textarea class="message" name="message" maxlength="500" placeholder="Write your message here"
+            required></textarea>
+        </li>
+        <li>
+          <button class="submit" type="submit">Get it touch</button>
```
Comment (prolajumokeoni) [inline index.html:231]: - [ ] Kindly add button interactions for when the submit button is pressed 👍🏿 

Screenshot_112

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Team
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR title and Description



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 - Kindly ensure that as partners you always have balanced commits in pair programming project 👍 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -1,195 +1,228 @@
...
+
+  <section class="form-background">
+    <h1 class="form-heading">Contact me</h1>
+    <p class="form-content">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it.</p>
+    <form class="contact-form" action="https://formspree.io/f/mvolvljr" method="POST">
+          <input class="name" type="text" name="firstname" maxlength="30" placeholder="Please enter your name "
+            required>
+          <input class="email" type="email" name="email" placeholder="Please enter your email" required>
+          <textarea class="message" name="message" maxlength="500" placeholder="Write your message here"
+            required></textarea>
```
Comment (gtekle) [inline index.html:222]: - [ Optional ] Please remove the black border in the input form elements and textarea as in the Figma design. You may try `border:none;` in the respective css rules.
- [Optional] The gap between the textarea and the button seems narrow in your implementation as compared to the design. kindly rectify.
contact-form-elements-impl

File: index.html [hunk 4]
```diff
@@ -1,195 +1,228 @@
...
+          <button class="submit" type="submit">Get it touch</button>
```
Comment (gtekle) [inline index.html:223]: - [ ] You should remove the submit button border as in the Figma design and as suggested by previous reviewer. You may try `border: none;` in the respective css rule
contact-form-elements-figma
.

File: index.html [hunk 5]
```diff
@@ -1,195 +1,228 @@
...
+  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
 </head>
+
 <body>
- 
+
```
Comment (gtekle) [inline index.html:16]: - [Optional ] Kindly remove the fixed sized width on website. You can always use developers tool to see the responsive layout.

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi @Khawanin :wave:, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
this-close


 ## Highlights
:heavy_check_mark: Good use of Github flow

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ Optional ] I believe the commit messages can be improved to be more descriptive. In general, commit messages should give you an idea on what has happened and where.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @Khawanin, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**